lua/neogen/configurations/python.lua

```diff
@@ -151,26 +144,24 @@ return {
                             end
                         end
 
+                        vim.pretty_print(res)
```

Comment:
```suggestion
```
Delete this, I think it is causing a message to pop up when i create a docstring